src/msg_recorder/recorder_config.h

```diff
@@ -3,13 +3,16 @@
 #include <simdjson.h>
 
 #include <chrono>
+#include <cstdint>
 #include <filesystem>
 #include <string>
 #include <vector>
 
 namespace cris::core {
 
 struct RecorderConfig {
+    enum class Rolling { NONE = 0, DAY = 1, HOUR = 2, SIZE = 3 };
```

Comment:
add trailing comma so that the formatter will put each item to a standalone line.